active/progress/progress.md

```diff
@@ -0,0 +1,107 @@
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+>See proposal documents for:
+>
+> [determinate mode for ProgressRing](https://github.com/microsoft/microsoft-ui-xaml/issues/688) 
+>
+> [ProgressRing style](https://github.com/microsoft/microsoft-ui-xaml/issues/837)
+>
+> [Guidance for Progress controls](https://github.com/microsoft/microsoft-ui-xaml/issues/880)
+
+Progress controls indicate to a user that an operation is occuring. This includes ProgressBar and ProgressRing controls. ProgressBar has both a determinate and indeterminate state. ProgressRing only has an indeterminate state.
+
+Important APIs: [ProgressBar](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.progressbar) and [ProgressRing](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.progressring)
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+The ProgressRing control indicates that an operation is ongoing. The determinate mode of ProgressRing shows the percentage completed of a task. 
+
+The style updates to progress controls will align with new designs and with other Microsoft products.
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+The following examples show how to use the isIndeterminate property to change the mode of the ProgressRing and the Value property to change the proportionate amount indicated. 
+
+## Indeterminate ProgressRing
+
+```xml
+<ProgressRing IsActive="True" Height="100" Width="100" IsIndeterminate="True"/>
+```
+
+## Determinate ProgressRing
+
+```xml
+<ProgressRing IsActive="True" Height="100" Width="100" Value="5"/>
+```
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+# API Notes
```

Comment:
ProgressRing derives from Control but ProgressBar derives from RangeBase. Is there value to changing ProgressRing's base class? And if there is value, is it possible to do so at this time?

Reply:
I think there is value in changing the base class to RangeBase given the addition of the determinate mode. I will determine if this is possible at this time.

Reply:
From team discussion it was decided that ProgressRing won't inherit from RangeBase, as ProgressBar does not use many properties it inherits from RangeBase.